### ISSUE TITLE
Wrong variable?

### ISSUE THREAD

**Issue by knokknok** [jipingw/DegNorm#6]
https://github.com/jipingw/DegNorm/blob/2e3e9c950849869289b8651f8c735aba4774cff5/R/coverage_cal_functions.R#L33

Should be
 ```
       if(!file.exists(paste(bam_file_list[k],".bai",sep=""))){
```
?

**Comment by jipingw** [comment]
sample_names were simplified version of bam_file_list names after stripping out those characters for file path/extension etc. the code should be fine.

**Comment by knokknok** [comment]
> sample_names were simplified version of bam_file_list names after stripping out those characters for file path/extension etc.

Yes, therefore the files `paste(sample_names[k],".bai",sep="")` will never exist and the bam files reindexed systematically.
